specification/resources/resource-manager/readme.md

```diff
@@ -535,6 +535,10 @@ directive:
     suppress: OperationsAPIImplementation
     where: $.paths
     reason: OperationsAPI will come from Resources
+  - from: deploymentScripts.json
+    suppress: R3006 #BodyTopLevelProperties
+    where: $.definitions.AzureResourceBase
+    reason: Currently systemData is not allowed
```

Comment:
@raych1 `systemData` is a valid top-level property now ([https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources)). Can we file a bug for the linter to stop complaining about that?

Reply:
the linter has fixed it in new version, and will be applied soon ,
currently, the places you suppressed are incorrect,
it should be like this :
 - from: deploymentScripts.json
    suppress: R3006 #BodyTopLevelProperties
    where: 
    - $.definitions.DeploymentScript.properties
    - $.definitions.AzureCliScript.properties
    - $.definitions.AzurePowerShellScript.properties
    reason: Currently systemData is not allowed